mod/wikirate/assets/style/about_pages.scss

```diff
@@ -357,3 +352,16 @@
     }
   }
 }
+
+.slick-current {
+  .nodblclick {
+      background-color: #6B5CD8;
+      color: #fff;
```

Comment:
can you use `$white` instead of `#fff`?

Currently those are the same, but if we use variables, then someone can change it to a different variant of the color and it will take effect everywhere.